doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -432,28 +445,36 @@ When you have all the source files that you need, run the configure script, whic
 ```
 bash configure --disable-ccache \
                --with-boot-jdk=/cygdrive/c/<path to_jdk8> \
-               --with-freemarker-jar=/cygdrive/c/temp/freemarker.jar \
                --with-freetype-src=/cygdrive/c/temp/freetype
 ```
 
 2) Win 32-bit
 ```
 bash configure --disable-ccache \
                --with-boot-jdk=/cygdrive/c/<path_to_jdk8> \
-               --with-freemarker-jar=/cygdrive/c/temp/freemarker.jar \
```

Comment:
We're getting a little ahead of ourselves here: cmake is not yet supported for win32: Should this change wait?

Reply:
Just a little ahead I hope, the cmake support for win32 seems just about ready to merge. I've restored the freemarker line for now. Also pushed changes to address all the other comments.